<?php

namespace ContainerA3rXPsa;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/*
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getValidator_Mapping_Cache_AdapterService extends App_KernelTestContainer
{
    /*
     * Gets the private 'validator.mapping.cache.adapter' shared service.
     *
     * @return \Symfony\Component\Cache\Adapter\PhpArrayAdapter
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['validator.mapping.cache.adapter'] = \Symfony\Component\Cache\Adapter\PhpArrayAdapter::create(($container->targetDir.''.'/validation.php'), ($container->privates['cache.validator'] ?? $container->load('getCache_ValidatorService')));
    }
}
